_content/doc/articles/wiki/final-noclosure.go

```diff
@@ -2,16 +2,17 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build ignore
```

Comment:
not sure if it's OK to add it. VS code did it for me. Should I leave it as it is?